faces.py

```diff
@@ -188,5 +208,12 @@ def get_random_person():
         'middle_name': x[2],
         'last_name': x[3],
         'image_url': x[4],
-        } for x in random_person]
+        'stints': [{
+            'stint_type': s[0],
+            'start_date': s[1],
+            'end_date': s[2],
+            'title': s[3],
+            'short_name': s[4]
+            } for s in stints],
```

Comment:
You're right - ideally, this would be refactored after a future update to separate the queries for getting a target person from getting three first names to display as hints.

In an earlier version, I had an if statement that only populated the stints for the target person at index 0, but I don't see it in this commit. I'll restore this filter to reduce a bit of the clutter.